src/modules/resource/reslog.c

```diff
@@ -136,14 +143,24 @@ int reslog_post_pack (struct reslog *reslog,
 
     if (!event)
         return -1;
-    if (!(val = eventlog_entry_encode (event)))
-        goto error;
-    if (!(txn = flux_kvs_txn_create ()))
-        goto error;
-    if (flux_kvs_txn_put (txn, FLUX_KVS_APPEND, RESLOG_KEY, val) < 0)
-        goto error;
-    if (!(f = flux_kvs_commit (reslog->h, NULL, 0, txn)))
-        goto error;
+    if ((flags & EVENT_NO_COMMIT)) {
+        if (!(f = flux_future_create (NULL, NULL)))
+            goto error;
+        flux_future_set_reactor (f, flux_get_reactor (reslog->h));
+        flux_future_set_flux (f, reslog->h);
```

Comment:
Doesn't `flux_future_set_flux(3)` do the equivalent of `flux_future_set_reactor (f, flux_get_reactor (reslog->h));`?

Reply:
Oh you're right, good catch!